frontend/public/components/deployment.jsx

```diff
@@ -117,7 +127,63 @@ const DeploymentsDetailsPage = props => <DetailsPage
 />;
 
 const Row = props => <WorkloadListRow {...props} kind="Deployment" actions={menuActions} />;
-const DeploymentsList = props => <List {...props} Header={WorkloadListHeader} Row={Row} />;
+
+const kind = 'Deployment';
+
+const DeploymentTableRow = (o, index) => {
+  return {
+    id: index,
```

Comment:
We should be using `metadata.uid` when possible for `id`

Reply:
noted - will update